benchmarks/viscoelastic_sheared_torsion/viscoelastic_sheared_torsion.prm

```diff
@@ -0,0 +1,169 @@
+# t = 0.5. At this point the shearing velocity is reduced to zero, and elastic stresses relax.
+# For t < 0.5 the analytical expression of the stress is given by:
+#   tau = exp(-mu/eta*t)*(C2*cos(V*t/h)-C1*sin(V*t/h))-C2
+# and for t >= 0.5:
+#   tau = exp(-mu/eta*tmax)*(C2*cos(V*tmax/h)-C1*sin(V*tmax/h))-C2)*exp(-mu/eta*(t-tmax))
+# with:
+#   tmax = 0.5 [s]     # time at which shearing stops
+#   eta = 100  [Pa s]  # the shear viscosity
+#   mu  = 100  [Pa]    # the shear modulus
+#   V = 0.3    [m/s]   # the shearing velocity
+#   h = 1      [m]     # the height of the box
+#   omega = 42         # the dimensionless rotation rate
+#   C1 = -(V^2*eta^2*mu)/(mu^2*h^2+V^2*eta^2)
+#   C2 = -(V*h*eta*mu^2)/(mu^2*h^2+V^2*eta^2)
+#
+# These analytical stresses can be directly compared to the xz-component of the stress output from this parameter file.
+# Farrington et al., however, look at the stresses in an unrotated frame (their figure 1b), in which case both,
+# the analytical stress and the aspect stress component need to be multiplied with cos(omega*t). Note that in the
+# Farrington paper the vertical axis is the y-axis, such that their stress component xy is aspect's stress
+# component xz.
+# The fit of the numerical stress to the analytical solution can be enhances by choosing a smaller time step size.
+
+
+# Global parameters
+set Dimension                                  = 3
+set Start time                                 = 0
+set End time                                   = 2
+set Use years in output instead of seconds     = false
+set Resume computation                         = auto
+set CFL number                                 = 1.0
+set Maximum time step                          = 0.01
+set Output directory                           = output
+set Pressure normalization                     = surface
+set Surface pressure                           = 0.
+set Nonlinear solver scheme                    = single Advection, iterated Stokes
+set Max nonlinear iterations                   = 30
+set Nonlinear solver tolerance                 = 1e-5
+set Max nonlinear iterations in pre-refinement = 0
+
+
+# Solver settings
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Number of cheap Stokes solver steps = 200
+    set Linear solver tolerance             = 1e-9
+  end
+end
+
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X extent  = 1.0
+    set Y extent  = 1.0
+    set Z extent  = 1.0
+    set X repetitions = 5
+    set Y repetitions = 5
+    set Z repetitions = 5
+    set Box origin X coordinate = -0.5
+    set Box origin Y coordinate = -0.5
+  end
+end
+
+subsection Mesh refinement
+  set Strategy                           = maximum refinement function
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+  subsection Maximum refinement function
+    set Function expression              = 0
+  end
+end
```

Comment:
The refinement strategy doesn't seem to do anything to the mesh (Function expression is zero). Can you clean this up?